src/components/BookDetail.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+function BookDetail() {
+  const books = useSelector((state) => state.books);
+
+  const { name } = useParams();
+  const findBook = books.books.find((book) => book.name === name);
+
+  if (!findBook) {
+    return <h2>Book not found</h2>;
+  }
+
+  return (
+    <>
+      <div className="main">
```

Comment:
Mind that there seems to be a overflow issue where the design does change styling when on certain thresholds, but its not being responsive by fitting inside the given screen or shrinking its size it it must do so 
![image](https://github.com/leloufadel/front-end-capstone/assets/91759364/af5c2e89-02db-4e23-bc53-5329e510d0da)

Reply:
Hello @Arch-Noize  thanks for your nice review but see for me it show like this 
![Uploading image.png…]()